packages/cli/src/cmds/preview/preview.ts

```diff
@@ -685,11 +685,11 @@ export default class Preview extends YargsCommand {
       sideloadingUrl = sideloadingUrl.replace(constants.accountHintPlaceholder, "");
     }
 
-    const sideloadingBar = CLIUIInstance.createProgressBar(constants.sideloadingTitle, 1);
-    await sideloadingBar.start(`${constants.sideloadingStartMessage}`);
+    const sideloadingBar = CLIUIInstance.createProgressBar(constants.previewTitle, 1);
```

Comment:
Do we need to update the variable name as well?

Reply:
Sorry, I will change it.